### DISCUSSION TITLE
RTSP Compressing & Saving stream to disk gets timed out.

### DISCUSSION THREAD

**Discussion by ebala96** [bluenviron/mediamtx#757] [Questions]
Hi,

I am trying to compress the RTSP stream. 
I followed to steps edited my yml file from https://github.com/aler9/rtsp-simple-server#remuxing-re-encoding-compression.

Once I play the stream using the compressed path, the connection gets timed out after 10 seconds.
[RTSP] [conn 127.0.0.1:32922] closed (source of path 'compressed' has timed out)

I am using VLC to play the stream.

What am I missing? Or should I add anything to my conf file?

**Comment by aler9** [comment]
Hello, make sure that ffmpeg is correctly installed on your system.

[a comment by an automated account is not shown]
